package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@vonage/vivid-react",
-  "version": "2.10.92",
+  "version": "2.10.93",
```

Comment:
did you generated this with the yarn command or manually pushed?

in this project usually version is updated with the yarn command which is also tagging

Reply:
done